lib/src/main/java/com/basistheory/android/service/CardBrandEnricher.kt

```diff
@@ -0,0 +1,215 @@
+package com.basistheory.android.service
+
+class CardBrandEnricher {
+
+    object CardMasks {
+        const val MASK_4_8_12GAPS_19LENGTH = "#### #### #### #######"
+        const val MASK_4_8_12GAPS_16LENGTH = "#### #### #### ####"
+    }
+
+    class CardDetails(
+        var brand: String = "",
+        var identifierRanges: List<Pair<String, String?>>,
+        var validLengths: IntArray,
+        var cvcMask: String = "",
+        var cardMask: String = ""
+    )
+
+    class CardResult(
```

Comment:
we could update this to only include brand, the masks and the complete flag, since the ranges and valid lengths shouldn't be concerning to the element calling this